lib/step-model-merger/types.ts

```diff
@@ -37,4 +37,10 @@ export interface MergeStepModelOptions {
   repo: Repository
   circuitJson: CircuitJson
   boardThickness: number
+  /**
+   * Pre-loaded STEP file contents, keyed by URL/path.
+   * If a URL is found here, the content is used directly instead of fetching.
```

Comment:
remove the ability for urls to appear here, a url should not appear in the fsMap (the comment is confusing)